app/src/main/java/ru/netology/fmhandroid/dto/Note.kt

```diff
@@ -10,6 +10,13 @@ data class Note(
     val planeExecuteDate: String,
     val factExecuteDate: String,
     val statusId: Int,
+    val status: NoteStatusEnum,
     var comment: String,
     val deleted: Boolean
-)
\ No newline at end of file
+)
+
+enum class NoteStatusEnum {
+    ACTIVE,
+    CANCELED,
+    EXECUTE
+}
```

Comment:
Так покороче будет. Плюс с именем основного класса не разойдётся название енума. Плюс суффикс `...Enum` впринцие лишнее уточнение для енума.

```suggestion
) {

    enum class Status {
        ACTIVE,
        CANCELED,
        EXECUTE
    }
}
```